hugegraph-core/src/main/java/com/baidu/hugegraph/k8s/K8sDriverProxy.java

```diff
@@ -111,15 +121,22 @@ protected void initConfig(String partitionsCount,
         this.config = new HugeConfig(mapConfig);
     }
 
-    protected void initKubernetesDriver() {
-        this.driver = new KubernetesDriver(this.config);
+    /**
+     * Reuse K8s driver for task to operate, init only if it was null
+     * TODO: use singleton for it
+     */
+    private void initK8sDriver() {
+        if (driver == null) {
+            driver = new KubernetesDriver(this.config);
+        }
     }
 
-    public KubernetesDriver getKubernetesDriver() {
-        return this.driver;
+    public KubernetesDriver getK8sDriver() {
+        return driver;
     }
 
     public void close() {
-        this.driver.close();
+        // TODO: Comment now & delete this method after ensuring it
+        //driver.close();
```

Comment:
when to close？maybe close when HugeGraphServer shutdown

Reply:
no need to close until server shutdown (remove this method after ensure it)